Classes/Domain/Index/Queue/RecordMonitor/Helper/ConfigurationAwareRecordService.php

```diff
@@ -123,37 +123,20 @@ protected function getRecordIfIndexConfigurationIsValid(
     protected function getRecordForIndexConfigurationIsValid(
         string $recordTable,
         int $recordUid,
-        string $recordWhereClause = '',
+        string $recordWhereClause = ''
     ): array {
-        $cache = GeneralUtility::makeInstance(TwoLevelCache::class, 'runtime');
+        $cache = GeneralUtility::makeInstance(TwoLevelCache::class, /** @scrutinizer ignore-type */ 'runtime');
```

Comment:
```suggestion
        $cache = GeneralUtility::makeInstance(TwoLevelCache::class, 'runtime');
```
We do not use scutinizer since c2558c1d3ab8fb8d04eac9213ee6e83538119ffb anymore.
Please remove `/** @scrutinizer ignore-type */`